ui/src/components/auth/SignInForm.tsx

```diff
@@ -1,19 +1,21 @@
-import { Alert, Box, Button, Link, TextField } from "@mui/material";
-import NextLink from "next/link";
-import React, { useState } from "react";
+import { Alert, Box, Button, TextField } from "@mui/material";
+import { FormEvent, useState } from "react";
 
-interface SignInFormProps {
-  onSubmit: (email: string, password: string) => void;
+import Link from "@/components/Link";
+import { SignInWithPasswordMutationVariables } from "@/graphql/generated";
+
+type SignInFormProps = {
+  onSubmit: (args: SignInWithPasswordMutationVariables) => void;
   errors?: string[];
-}
+};
 
-const SignInForm: React.FC<SignInFormProps> = ({ onSubmit, errors }) => {
```

Comment:
There's [not really any reason](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/) to use `React.FC`, removed. Also function declaration with default export for consistency